### ISSUE TITLE
SQLITE_ERROR

### ISSUE THREAD

**Issue by TheRealHotSwap** [3s3s/opentrade#187]
SELECT ERROR: query=SELECT ROWID AS id, info FROM users WHERE id=undefined message=SQLITE_ERROR: no such column: undefined

Should there be an ID section added to 'users' ?

      'name' : 'users',
      'cols' : [
          ['login', 'TEXT UNIQUE'],
          ['email', 'TEXT UNIQUE'],
          ['password', 'TEXT'],
          ['info', 'TEXT']
        ],

**Comment by jonn4y** [comment]
no, SELECT ROWID AS id is selecting the ROWID in the database and calling it ID in the SQL command. 

your error is you haven't passed a ID to the WHERE function

WHERE id=undefined 

check where you are passing a id in your code

**Comment by TheRealHotSwap** [comment]
It's right when I start up the exchange. Everything seems to work fine but on occasion, the order books disappear but that doesn't seem to be related.

@jonn4y I do appreciate your help, thanks!
